examples/video_mode/test_user_input.py

```diff
@@ -0,0 +1,80 @@
+from matplotlib import pyplot as plt
+from qm.qua import *
+from qm import QuantumMachinesManager
+from qualang_tools.results import fetching_tool
+from configuration import *
+from qualang_tools.video_mode.videomode import VideoMode
+
+
+def qua_prog(vm: VideoMode):
+    with program() as prog:
+        # Results variables
+        single_shot_1 = declare(fixed)
+        single_shot_2 = declare(fixed)
+        # Get the parameters from the video mode
+        dc_offset_1, dc_offset_2 = vm.declare_variables()
+        # Streams
+        signal1_st = declare_stream()
+        signal2_st = declare_stream()
+
+        with infinite_loop_():
+            # Update the parameters
+            vm.load_parameters()
+            # Update the dc_offset of the channel connected to the OPX analog input 1
+            set_dc_offset("filter_cavity_1", "single", dc_offset_1)
+            set_dc_offset("filter_cavity_2", "single", dc_offset_2)
+            # Measure and integrate the signal received by the OPX
+            measure(
+                "readout",
+                "detector_DC",
+                None,
+                integration.full("constant", single_shot_1, "out1"),
+                integration.full("constant", single_shot_2, "out2"),
+            )
+            # Save the measured value to its stream
+            save(single_shot_1, signal1_st)
+            save(single_shot_2, signal2_st)
+            # Wait between each iteration
+            wait(1000)
+
+        with stream_processing():
+            signal1_st.buffer(1000).save("signal1")
+            signal2_st.buffer(1000).save("signal2")
+    return prog
+
+
+if __name__ == "__main__":
+    # Open the Quantum Machine Manager
+    qmm = QuantumMachinesManager(qop_ip, cluster_name=cluster_name)
+    # Open the Quantum Machine
+    qm = qmm.open_qm(config)
+    # Define the parameters to be updated in video mode with their initial value
+    param_dict = {
+        "dc_offset_1": 0.0,
+        "dc_offset_2": 0.0,
+    }
+    # Initialize the video mode
+    video_mode = VideoMode(qm, param_dict)
+    # Get the QUA program
+    qua_prog = qua_prog(video_mode)
+    # Execute the QUA program in video mode
+    job = video_mode.execute(qua_prog)
```

Comment:
I don't see anywhere in the code that the parameters are being updated, don't we want to show that?

Reply:
The functionality of the VideoMode is that the parameters are loaded by the user through his terminal or the Python console (meaning he has to type them down once the program is already started). The main objective of this mode is to enable a change of parameters without knowing their values in advance while the program runs in the background. Everything happens therefore with user input() Python function which is handled behind the scenes in the class. There's no parameter update to show in the script since the user is summoned to update them once the program is launched (through print statements and waiting for input in the console, while the QUA program runs in the background)

Reply:
Ok, I know understand you take over the console.
This isn't explained anywhere, but should appear in the readme and not here